src/librustc_trans/trans/common.rs

```diff
@@ -547,7 +552,7 @@ impl<'a, 'tcx> FunctionContext<'a, 'tcx> {
 // code.  Each basic block we generate is attached to a function, typically
 // with many basic blocks per function.  All the basic blocks attached to a
 // function are organized as a directed graph.
-pub struct BlockS<'blk, 'tcx: 'blk> {
+pub struct BlockS {
```

Comment:
@eddyb Yeah, I figured... perhaps something like `BlockData` would be better?  Or, rename it to `Block` and call the thing we're passing around everywhere `BlkCx` or something (we even call it `bcx` everywhere).